Extension/package.json

```diff
@@ -1630,9 +1630,9 @@
   },
   "scripts": {
     "vscode:prepublish": "yarn run compile",
-    "compile": "yarn run precompile && webpack --mode production --vscode-nls",
+    "compile": "yarn run precompile && yarn run translations-generate && webpack --mode production --vscode-nls",
     "compile-dev": "yarn run precompile && webpack --mode development",
-    "compile-watch": "yarn run precompile && webpack --mode production --vscode-nls --watch --info-verbosity verbose",
+    "compile-watch": "yarn run precompile && yarn run translations-generate && webpack --mode production --vscode-nls --watch --info-verbosity verbose",
     "compile-dev-watch": "yarn run precompile && webpack --mode development --watch --info-verbosity verbose",
```

Comment:
This could also go in precompile (not sure what is preferred).

Reply:
I don't think this is a good idea.  compile-watch is not for a production build, so shouldn't incur the overhead of generating translations.  If you need translations, use the production build.

Reply:
Er, it looks like it's needed on compile, but I'd prefer to don't add it to compile-watch.

Reply:
Ok